Tests/CommandsTests/RunToolTests.swift

```diff
@@ -86,12 +86,38 @@ final class RunToolTests: XCTestCase {
         }
     }
 
+    // Test that thread sanitizer works.
+    func testSanitizeThread() throws {
+        // FIXME: We need to figure out how to test this for linux.
+      #if os(macOS)
+        fixture(name: "Miscellaneous/ThreadRace") { path in
+            // Ensure that we don't abort() when we find the race. This avoids
+            // generating the crash report on macOS.
+            let env = ["TSAN_OPTIONS": "abort_on_error=0"]
```

Comment:
Why `abort_on_error=0`? We do expect it to abort asap, no?

Reply:
The default option makes the program to call abort() on first issue. This causes the program to generate a crash report on macOS, which is bad because every time we run this test, a new report will be generated.